test/tasksqueu.test.ts

```diff
@@ -0,0 +1,22 @@
+import Database from '../src/database';
+import { Post, TasksQueue } from '../src/app';
```

Comment:
```suggestion
import { TasksQueue } from '../src/app';
```
Post здесь не юзается